common/components/NavigationLink.tsx

```diff
@@ -8,13 +8,18 @@ interface Props extends RouteComponentProps<{}> {
   link: NavigationLink;
   isHomepage: boolean;
   className: string;
+  isNotEnabled: boolean;
```

Comment:
Can we make this an optional boolean? If its not set, it should be falsy anyway.